Examples/Physics_applications/laser_acceleration/inputs_1d

```diff
@@ -70,3 +70,13 @@ diagnostics.diags_names = diag1
 diag1.intervals = 100
 diag1.diag_type = Full
 diag1.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz rho
+
+# Reduced Diagnostics
+warpx.reduced_diags_names               = FP
+
+FP.type = FieldProbe
+FP.intervals = 10
+FP.integrate = 0
+FP.probe_geometry = Point
+FP.z_probe = 0
```

Comment:
Oh, can you please do a line here, too?
1D means there is only a 1D line geometry modeled in z :)

Or to be precise: we assume everything in x and y is symmetric/periodic, basically like a long line of flat capacitors with different densities interacting with each other.

(See my https://doi.org/10.5281/zenodo.3266820, Figure 2.10 picture "1D3V").